src/components/scenes/LoginScene.tsx

```diff
@@ -81,6 +82,7 @@ class LoginSceneComponent extends React.PureComponent<Props, State> {
       .then(backgroundImage => this.setState({ backgroundImage }))
       // @ts-expect-error
       .catch(e => this.setState({ backgroundImage: theme.backgroundImage }))
+    makeStakePlugins().catch()
```

Comment:
The empty `catch` does nothing. You need to do `catch(() =>{})` to throw the error away, plus maybe a comment that `// This is just here to load the cache`.

Reply:
Also, I do think this would be better in the services, but it's fine either place.

Reply:
I agree so I'll move it there